src/components/forms/NetworkingRulesOrderCard.js

```diff
@@ -245,3 +258,81 @@ NetworkingRulesOrderCard.propTypes = {
 };
 
 export default NetworkingRulesOrderCard;
+
+const OrderCardDataTable = (props) => {
```

Comment:
not in love with having the data table and then the networking rule edit forms. can we combine the two? for example, 
- make the names in the data table clickable and on click, display the networking rule modal form for editing 
- add the up/down arrows to the data table

Reply:
i agree, to me, both of them is a bit ugly. i think this is doable but it is very inconsistent with the rest of the patterns in craig though, which concerns me a bit

Reply:
not actually sure the up down arrows is possible but i think it might be